main.py

```diff
@@ -293,10 +308,10 @@ def sell():
                 for coin in list(order):
 
                     # store some necesarry trade info for a sell
-                    stored_price = float(coin['price'])
+                    stored_price = float(coin['fills'][0]['price'])
```

Comment:
orders can be filled with multiple fills so better use the average price in the response